recipes/xerces-c/all/CMakeLists.txt

```diff
@@ -0,0 +1,22 @@
+cmake_minimum_required(VERSION 2.8.12)
+project(cmake_wrapper)
+
+include(conanbuildinfo.cmake)
+conan_basic_setup()
+
+if("${CMAKE_CXX_COMPILER_ID}" STREQUAL "GNU")
+  add_definitions("-Wno-cast-qual")
```

Comment:
Just curious: Why are all of those flags needed depending on the compiler? Is this kind of a patch?

Reply:
@SSE4 could you please remind why this was added? -Werror via option `fatal-warnings` is turned off by default afaik.

Reply:
we had to add such lines to silence some warnings. travis CI limits the size of log, but such warnings were so frequent that they caused travis to fail the build. ignoring them is harmless and doesn't affect the package ID at all. for conan-center-index, I guess, it could be removed (unless, conan-center-index has a limit for log size as well)

Reply:
Thanks, got it. I'll try to remove them if CCI handles large log files.
Regarding the large test – I can make a typical test that checks only library loading and function calls, it is enough to test packaging.